persistence/registry/gormRegistry.go

```diff
@@ -0,0 +1,28 @@
+package registry
+
+import (
+	"github.com/ditrit/badaas/persistence/gormdatabase"
+	"github.com/ditrit/badaas/persistence/models"
+	"github.com/ditrit/badaas/persistence/repository"
+)
+
+// The list of database table to migrate if necessary
+var listOfDatabaseTables = []any{
```

Comment:
I'm not fan to have this variable here, is that possible to get dynamically  all struct in a specicif folder and instanciate them ?

Reply:
Simple answer: no.

Complicated answer: we could create some tooling to do this but it would be non standard